content/kb/tutorials/deploy/docker.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Deploy Streamlit using Docker
+slug: /knowledge-base/tutorials/deploy/docker
+---
+
+# Deploy Streamlit using Docker
+
+## Introduction
+
+So you have an amazing app and you want to start sharing it with other people, what do you do? You have a few options. First, where do you want to run your Streamlit app, and how do you want to access it?
+
+- **On your corporate network** - Most corporate networks are closed to the outside world. You typically use a VPN to log onto your corporate network and access resources there. You could run your Streamlit app on a server in your corporate network for security reasons, to ensure that only folks internal to your company can access it.
+- **On the cloud** - If you'd like to access your Streamlit app from outside of a corporate network, or share your app with folks outside of your home network or laptop, you might choose this option. In this case, it'll depend on your hosting provider. We have community-submitted guides from Heroku, AWS, and other providers [here](/knowledge-base/deploy/deploy-streamlit-heroku-aws-google-cloud).
```

Comment:
Can you use meaningful [link text](https://developers.google.com/style/link-text)?

Reply:
Nice! I didn't know there was a best practice 😃